modules/DCC/Portal.yaml

```
@@ -0,0 +1,109 @@
+enums:
+  DataStatusEnum:
+    title: Data Status
+    permissible_values:
+      Data Not Expected: 
+        description: Data is not expected for the project.
+      Data Pending: 
+        description: There is no data yet in the project because it is still being generated or has not yet been uploaded yet.
+      Under Embargo: 
+        description: Data is present in the project but not accessible to anyone outside the project admins. 
+        notes: 
+        - When data is first uploaded, the status should change from “Data Pending” to “Under Embargo”.
+      Rolling Release: 
+        description: Some data is available for download for the project via rolling release.
+      Partially Available: 
+        description: Some data is available for download for the project.
+      Available: 
+        description: Data is fully available for download for the project (with fulfillment of any governance requirements).
+
+  
+  StudyStatusEnum:
+    title: Study Status
+    permissible_values:
+      Active: 
+        description: The project is in the performance period between grant start and grant end dates.
+      Completed: 
+        description: The project has reached the grant end date and all intended data has been generated and uploaded.
+      Withdrawn: 
+        description: The project was planned/started but not completed (withdrawn for various reasons).
+
+  DiseaseFocusEnum:
+    title: Disease Focus
+    permissible_values:
+      Neurofibromatosis type 1:
+      Neurofibromatosis type 2:
+      Multiple:
```

```suggestion
      Schwannomatosis:
      Multiple:
```

NF2 is now known as NF2-related schwannomatosis, and what we in the past called "schwannomatosis" is now SMARCB1-related schwannomatosis _or_ LZTR-related schwannomatosis depending on the variants. Is it time to rip off the bandaid and make this change portal wide? 

See pp6 here https://www.ctf.org/wp-content/uploads/2023/12/CTF-Diagnosed-with-NF2SWN-Brochure.pdf

---

Yeah, changes like this can be painful, but I suppose it's an improvement to make things clearer/more correct. Also if CTF appreciates being consistent, we'll of course do it. But it should be "NF2-related schwannomatosis" instead of just "Schwannomatosis"?

---

Oops, I should have made two comments because I was kind of combining a code suggestion (at a minimum, let's add Schwannomatosis) and a discussion (maybe we should just update the NF2 and Schwannomatosis related values entirely to reflect the new diagnostic categories. Previous diagnoses in the portal of Schwannomatosis should be marked as Schwannomatosis-NOS *or *we can manually check whether they should be SMARCB1-related schwannomatosis, LZTR1-related schwannomatosis, etc. I think this shouldn't be too much of a lift because I think the only project this applies for is Synodos Schwannomatosis